### PR TITLE
Adding Prefix and Group tags

### PR DESCRIPTION
**Pull request by AdrianWennberg**
Prefix and Group tags added for RPMs only since those tags do not exist for deb packages. 

Also defined some macros for rpmbuild to make nfpm less dependent on the build environment.

### CODE REVIEW

**Comment by caarlos0** [comment]
any thing like this for deb?

if its only for RPM, maybe prefix the config with `RPM` or inside a RPM block to make it clearer 🤔

**Comment by AdrianWennberg** [comment]
The `Section` tag which is already implemented for deb-only seems to be similar to the `Group` tag for RPMs, but it does not make sense to use one tag for both since they have different naming conventions. 

For reference:
RPM packages by group: https://www.rpmfind.net/linux/RPM/Groups.html
Deb packages by section: https://packages.debian.org/stable/

Putting the new tags into an `RPM` block seems like a good solution to avoid confusion.

**Comment by caarlos0** [comment]
Yeah seems good to me! And the deb specific ones we can put under a Deb block too..

willing to make this change?

**Comment by AdrianWennberg** [comment]
Moving the `Section` tag under a Deb block might break current config files if people update nfpm without updating their config. Do you think this will be an issue for people?

Other that that, I'd be happy to make the change!

**Comment by AdrianWennberg** [comment]
Oops forgot to actually create the package in the test.

**Comment by AdrianWennberg** [comment]
Hey Carlos,

I think this is merge ready now. Give me a shout if there is anything else you would like me to add.

File: rpm/rpm_test.go [hunk 1]
```diff
@@ -94,6 +94,16 @@ func TestRPM(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestWithRPMTags(t *testing.T) {
+	var info = exampleInfo()
+	info.RPM = nfpm.RPM{
+		Group:  "default",
+		Prefix: "/usr",
+	}
+	var err = Default.Package(info, ioutil.Discard)
```
Comment (caarlos0) [inline rpm/rpm_test.go]: maybe write to a file and compare with the expected result? (like goldenfiles above)

Comment (AdrianWennberg) [reply rpm/rpm_test.go]: I added a new commit. Is that what you had in mind?

Comment (caarlos0) [reply rpm/rpm_test.go]: looks great, thanks

**Comment by caarlos0** [comment]
thank you very much for the PR and all the effort :) ✨

**Comment by caarlos0** [comment]
This pull request has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.

**Pull request merged**